auth/src/main/res/layout-land/fui_auth_method_picker_layout.xml

```diff
@@ -34,4 +34,13 @@
 
     </ScrollView>
 
+    <TextView
+        android:id="@+id/main_tos_and_pp"
+        style="@style/FirebaseUI.Text.BodyText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/fui_field_padding_vert"
+        android:textColor="?android:textColorTertiary"
```

Comment:
What if you do `?attr:textColorTertiary` does it look the same?  If so, prefer that form since it will be more theme-able.